src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.tsx

```diff
@@ -303,101 +275,101 @@ function BaseValidateCodeForm(props) {
-                        ref={inputValidateCodeRef}
-                        label={props.translate('common.magicCode')}
+                        autoComplete={autoComplete}
+                        ref={(magicCodeInput) => {
+                            inputValidateCodeRef.current = magicCodeInput;
+                        }}
                         name="validateCode"
                         value={validateCode}
                         onChangeText={(text) => onTextInput(text, 'validateCode')}
                         onFulfill={validateAndSubmitForm}
-                        errorText={formError.validateCode ? props.translate(formError.validateCode) : ''}
+                        errorText={formError.validateCode ? translate(formError.validateCode) : ''}
                         hasError={hasError}
                         autoFocus
                         key="validateCode"
                         testID="validateCode"
                     />
-                    {hasError && <FormHelpMessage message={ErrorUtils.getLatestErrorMessage(props.account)} />}
+                    {hasError && !isEmptyObject(account) && <FormHelpMessage message={ErrorUtils.getLatestErrorMessage(account)} />}
                     <View style={[styles.alignItemsStart]}>
-                        {timeRemaining > 0 && !props.network.isOffline ? (
+                        {timeRemaining > 0 && !isOffline ? (
                             <Text style={[styles.mt2]}>
-                                {props.translate('validateCodeForm.requestNewCode')}
+                                {translate('validateCodeForm.requestNewCode')}
                                 <Text style={[styles.textBlue]}>00:{String(timeRemaining).padStart(2, '0')}</Text>
                             </Text>
                         ) : (
                             <PressableWithFeedback
                                 style={[styles.mt2]}
                                 onPress={resendValidateCode}
-                                underlayColor={theme.componentBG}
                                 disabled={shouldDisableResendValidateCode}
                                 hoverDimmingValue={1}
                                 pressDimmingValue={0.2}
                                 role={CONST.ROLE.BUTTON}
-                                accessibilityLabel={props.translate('validateCodeForm.magicCodeNotReceived')}
+                                accessibilityLabel={translate('validateCodeForm.magicCodeNotReceived')}
                             >
-                                <Text style={[StyleUtils.getDisabledLinkStyles(shouldDisableResendValidateCode)]}>
-                                    {hasError ? props.translate('validateCodeForm.requestNewCodeAfterErrorOccurred') : props.translate('validateCodeForm.magicCodeNotReceived')}
+                                <Text style={[StyleUtils.getDisabledLinkStyles(shouldDisableResendValidateCode) as TextStyle]}>
```

Comment:
Update `getDisabledLinkStyles` type instead
```suggestion
                                <Text style={StyleUtils.getDisabledLinkStyles(shouldDisableResendValidateCode)}>
```